docs/source/nodes_and_pipelines/nodes.md

```diff
@@ -181,3 +181,155 @@ Out[2]: {'sum': 5}
 ```{note}
 You can also call a node as a regular Python function: `adder_node(dict(a=2, b=3))`. This will call `adder_node.run(dict(a=2, b=3))` behind the scenes.
 ```
+
+## How to use generator functions in a node
+
+Generator functions are useful for providing lazy iterators over big number of items without storing all of them in memory at once. We can wrap generator functions in a Kedro pipeline, the Kedro runner will make sure to save the result by calling the dataset's `save` method separately for each chunk.
+
+The following tutorial will show you how to utilise `pandas chunksize` generator to process large datasets, we will demonstrate this using the [`pandas-iris`](https://github.com/kedro-org/kedro-starters/tree/main/pandas-iris) [starter](https://kedro.readthedocs.io/en/stable/kedro_project_setup/starters.html):
```

Comment:
```suggestion
The following tutorial shows how to use a `pandas chunksize` generator to process large datasets within the [`pandas-iris`](https://github.com/kedro-org/kedro-starters/tree/main/pandas-iris) [starter](https://kedro.readthedocs.io/en/stable/kedro_project_setup/starters.html):
```

Reply:
@SajidAlamQB When you link to other pages of docs, you need to use a relative link rather than hard code the URL of the docs. I've corrected the one above but can you check your others please?

Reply:
Will do, thank you! 🙏

Reply:
I would then add a section which explains how to create the project. So I'd say something about the fact we assume you have already set up Kedro--something like this:

If you have not yet set up Kedro, do so by [following the guidelines to install Kedro](../get_started/install.md).)

Then, link to [the page that explains how to create a new project](../get_started/new_project.md#create-the-example-project) to show how to make the project.

Reply:
I will link to how to use `Kedro starters` as that should cover how to create the `pandas-iris` project properly.